v03_pipeline/lib/reference_datasets/mitomap.py

```diff
@@ -0,0 +1,29 @@
+import hail as hl
+
+from v03_pipeline.lib.model import ReferenceGenome
+
+
+# adapted from download_and_create_reference_datasets/v02/mito/write_mito_mitomap_ht.py
+def get_ht(raw_dataset_path: str, reference_genome: ReferenceGenome) -> hl.Table:
+    ht = hl.import_table(
+        raw_dataset_path,
+        delimiter=',',
+        types={'"Position"': hl.tstr, '"Allele"': hl.tstr},
```

Comment:
by default hail imputes everything as a string, so the only reason to explicitly pass types for imputation is if you want it to not be a string (although for position it does feel weird that you can't just import it directly as an int)